puppet/lib/puppet/face/node/status.rb

```diff
@@ -19,7 +19,7 @@
 
       args.map do |node|
         begin
-          response = Puppet::Util::Puppetdb::Http.action("/v4/nodes/#{CGI.escape(node)}") do |http_instance, path|
+          response = Puppet::Util::Puppetdb::Http.action("/pdb/query/v4/nodes/#{CGI.escape(node)}") do |http_instance, path|
```

Comment:
I'm a little surprised to see these kinds of changes in this PR. The switch to /pdb/query was made in another PR, right?